.github/workflows/deno-build.yml

```diff
@@ -0,0 +1,46 @@
+name: deno.land Continuous Deployment
+
+on:
+  push:
+    tags:
+      - '@slack/web-api*'
+
+defaults:
+  run:
+    shell: bash
+    working-directory: packages/web-api
+
+jobs:
+  transpile:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - name: Echo Tag Name
+      run: echo "$GITHUB_REF_NAME"
+    - uses: actions/checkout@v2
+      with:
+        # pull all tags/branches with full history;
+        # need that to checkout different branch and push artifacts later
+        fetch-depth: 0
+    - name: Set up Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 16
+    - name: Inject browser field and add dependencies to package.json
+      run: |
+        perl -pi.bak -e 's/"tsd": {/"browser":{"os":"os-browserify","path":"path-browserify","querystring":".\/deno-shims\/qs-shim.js"},"tsd": {/g' package.json
```

Comment:
@stevengill says maybe we could just add the `browser` field to the root package.json of the project.